### ISSUE TITLE
NEP-5 Tokens not being displayed... (0.2.2)

### ISSUE THREAD

**Issue by HODLyoda** [CityOfZion/neon-wallet#872]
My additional tokens (RPX, ONT, TKY) aren't showing up when I open the wallet or refresh the page.  They only show up after I successfully claim GAS.  I can't figure out how to get them to display otherwise.  Any help would be greatly appreciated.  (The version is 0.2.2, and I have the latest version of the NEO app on my Ledger Nano S.)

**Comment by Cryptick** [comment]
Same issue as @HODLyoda.  Neon Wallet version 0.2.2 with Ledger Nano S firmware 1.4.1.  Also I confirmed the Windows downloaded Neon Wallet hashfile before install.  No NEP-5 tokens are showing in my Neon Wallet.  I tried adding NEP5 tokens ONT & DBC using Manage Neon settings > Manage Tokens.  Tried restarting multiple times and still nothing.  The wallet does accept and does show my NEO and GAS balance.  I can claim GAS.  Also I tried removing and reinstalling the NEO app on my Ledger Nano S and still no NEP-5 tokens displayed.
